File: Leetcode/58_Length_of_Last_Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int ans = 0;
        for (int i = s.size()-1; i >= 0; i--) {
            if (s[i] == ' ') {
                if (ans > 0) return ans;
            }
            else ans++;
        }
        return ans;
    }
};

File: Leetcode/58_Length_of_Last_Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        cnt = 0
        for c in range(len(s)-1, -1, -1):
            if cnt != 0 and s[c] == ' ':
                break
            elif s[c] == ' ':
                continue
            else:
                cnt += 1
        return cnt